modules/t/citest_division/compara_citest.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<compara_db division="test">
+<compara_db division="citest">
+
+  <!-- Collections are species-sets that are needed to define several mlsss -->
+  <collections>
+
+    <!-- Mammals + Chicken -->
+    <collection name="mammals_with_feathers">
```

Comment:
Since "mammals_with_feathers" is only used in one place, I wouldn't define here in `<collections>` but directly in the EPO2X entry below